Face/Identify/Program.cs

```diff
@@ -0,0 +1,218 @@
+
+                    if (faces == null) 
+                    { 
+                        Console.WriteLine($"[Error] No persisted face from image `{targetImageFileName}`."); 
+                        return; 
+                    } 
+                } 
+            }
+
+            // Start to train the person group.
+            await Client.PersonGroup.TrainAsync(GroupId);
+
+            // Wait until the training is completed.
+            while (true)
+            {
+                await Task.Delay(1000);
+                var status = await Client.PersonGroup.GetTrainingStatusAsync(GroupId);
+                Console.WriteLine($"Response: Success. Group \"{GroupId}' training process is {status.Status}");
+
+                if (status.Status != TrainingStatusType.Running)
+                {
+                    break;
+                }
+            }
+
+            List<Guid> faceIds = new List<Guid>();
+            // Read image file. 
+            using (FileStream stream = new FileStream(Path.Combine("Images", SourceImageFileName), FileMode.Open)) 
+            { 
+                // Detect faces from image stream. 
+                IList<DetectedFace> detectedFaces = await Client.Face.DetectWithStreamAsync(stream); 
+                if (detectedFaces == null || detectedFaces.Count == 0) 
+                { 
+                    Console.WriteLine($"[Error] No face detected from image `{SourceImageFileName}`."); 
+                    return; 
+                } 
+
+                Console.WriteLine($"{detectedFaces.Count} faces detected from image `{SourceImageFileName}`."); 
+                if (detectedFaces[0].FaceId == null) 
+                { 
+                    Console.WriteLine("[Error] Parameter `returnFaceId` of `DetectWithStreamAsync` must be set to `true` (by default) for Identify purpose.");
+                    return; 
+                }
+
+                // Add detected faceId to faceIds.
+                foreach (var face in detectedFaces)
+                {
+                    faceIds.Add(face.FaceId.Value);
+                }
+            }
+
+            // Identify example for get person from the person group with candidate confidence. 
+            var identifyResults = await Client.Face.IdentifyAsync(faceIds, GroupId);
+            if (identifyResults == null)
+            {
+                Console.WriteLine($"No people found in the group the same as the {SourceImageFileName}.");
+                return;
+            }
+
+            foreach (var identifyResult in identifyResults)
+            {
+                Person person = await Client.PersonGroupPerson.GetAsync(GroupId, identifyResult.Candidates[0].PersonId);                
+                Console.WriteLine($"Person from {SourceImageFileName} & group person '{person.Name}' are of the same (Positive) person, confidence: {identifyResult.Candidates[0].Confidence}.");
```

Comment:
`Console.WriteLine($"Person '{person.Name}' is identified for faces in {SourceImageFileName}, confidence: {identifyResult.Candidates[0].Confidence}.");`

Reply:
done